src/dynamodb/DynamoDBManager.ts

```diff
@@ -115,6 +115,7 @@ export class DynamoDBManager {
       putItemInput.Item = {};
     }
 
+    putItemInput[this.config.conditionExpressionAttributeName] = putPointInput.ConditionExpression;
```

Comment:
why do you setup ConditionExpression in a const var conditionExpressionAttributeName ?
```
putItemInput. ConditionExpression = putPointInput.ConditionExpression;